hedera-node/log4j2.xml

```diff
@@ -140,6 +140,7 @@
         <MarkerFilter marker="FREEZE" onMatch="ACCEPT" onMismatch="NEUTRAL"/>
         <MarkerFilter marker="STALE_EVENTS" onMatch="DENY" onMismatch="NEUTRAL"/>
         <MarkerFilter marker="SNAPSHOT_MANAGER" onMatch="ACCEPT" onMismatch="NEUTRAL"/>
+        <MarkerFilter marker="EVENT_PARSER" onMatch="ACCEPT"  onMismatch="NEUTRAL"/>
```

Comment:
Shall the same marker be added to log4j2-prod.xml as well??

Reply:
not required, `EVENT_PARSER` used in state recover, which is done in offline mode